custom/enikshay/tasks.py

```diff
@@ -147,11 +195,14 @@ def update_single_case(self, episode_case):
         if update_json:
             update_case(self.domain, episode_case.case_id, update_json)
 
-    def _get_open_episode_cases(self):
+    def _get_case_ids(self):
         case_accessor = CaseAccessors(self.domain)
         case_ids = case_accessor.get_open_case_ids_in_domain_by_type(CASE_TYPE_EPISODE)
```

Comment:
Another way to split this up is that you could have one task per partitioned database. I think we have 10 on NIC/enikshay, but 5 on softlayer? (should check that to be sure though)